get-shit-done/workflows/feedback.md

```
@@ -0,0 +1,165 @@
+> "Title (≤72 chars): the question itself.
+> Description: relevant context (GSD version, command used, etc.)."
+
+Record `ISSUE_TITLE` and `ISSUE_BODY`.
+
+Note the current GSD version if available:
+```bash
+cat ~/.claude/get-shit-done/VERSION 2>/dev/null
+```
+
+If version is found, append to the body:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)